tests/test-setup.sh

```diff
@@ -0,0 +1,52 @@
+set -eu
+
+# */
+# To use envtest is required etcd, kube-apiserver and kubetcl binaries in the testbin directory.
+# This sript will perform this setup for linux or mac os x envs.
+# */
+
+K8S_VER=v1.18.2
```

Comment:
why so old?

Reply:
I forgot to delete this file, deleting it now